osgar/logger.py

```diff
@@ -146,8 +146,10 @@ def write(self, stream_id, data, dt=None):
         return dt
 
     def close(self):
-        self.f.close()
-        self.f = None
+        if self.f is not None:
```

Comment:
well, this is the place where I ask, or maybe I do not understand? (note, that logger is shared among multiple modules)

Reply:
The idea was move the if to the record. It could be maybe clearer.. but you can ignore it.